apis/elbv2/v1beta1/ingressclassparams_types.go

```diff
@@ -107,6 +109,10 @@ type IngressClassParamsSpec struct {
 	// +optional
 	SSLPolicy string `json:"sslPolicy,omitEmpty"`
 
+	// MutualAuthentication specifies the mutual TLS authentication config for all Ingresses that belong to IngressClass with this IngressClassParams.
+	// +optional
+	MutualAuthentication *MutualAuthentication `json:"mutualAuthentication,omitempty"`
```

Comment:
type MutualAuthentication struct {
 Mode ...
 TrustStoreArn...
}
type IngressClassParamsSpec struct {
	....

	// MutualAuthentication specifies the mutual TLS authentication config for all Ingresses that belong to IngressClass with this IngressClassParams.
	// +optional
	MutualAuthentication *MutualAuthentication `json:"mutualAuthentication,omitempty"`
}

Reply:
As discussed, removed the support for IngressClassParam from this release. We will add a documentation update which will specify that warning to use mTLS config on IngressGroups. Thanks for the review. 
We will add the IngressClassParam support for mTLS in follow up patch release.